src/components/NewsletterSignup/NewsletterSignup.mdx

```diff
@@ -72,6 +78,34 @@ _NOTE: This is applicable for all component props accepting HTML/JSX elements._
 
 <Canvas of={NewsletterSignupStories.DescriptionUsingJSXElements} />
 
+## Custom Title Heading
+
+The optional `title` prop will render the description list's heading as an `h2`
+element (sized as an `h3`) by default with text `"Sign Up for Our Newsletter"`.
+When the `title` value is a string, the default heading will be an `h2` but not
+sized as an `h3. It's possible that the `h2`element will not fit in an app's heading hierarchy. If this is the case,`title`can also take a`JSX.Element`
```

Comment:
`h3` is missing the closing tick mark. It also looks like there are spaces missing after the closing tick marks on some of the other words.